### ISSUE TITLE
Warrior's charge randomly doesn't stun the enemy

### ISSUE THREAD

**Issue by revar** [LightsHope/server#1041]
I tested it even with my friend on duel, and several times there was just no stun. No information in combat log about resist or anything.

**Comment by Cabro** [comment]
Mind posting the combat log next time? Or some video footage? It would help the devs a lot reproducing it.

**Comment by bigfgeg** [comment]
I've experienced this before and I found that it mostly has to do with lag. Chinese players are unstunnable by means of charge/intercept at times. 

Combat log will simply not show the charge/intercept at all, but it will go on cooldown and move you.

**Comment by revar** [comment]
Yeah but as i said the same problem happens in duel, sometimes there is info about being stunned and the stun fades in combat log and sometimes not. And we're both from eu, same country.

Will try to record it and post it here soon.

**Comment by GuybrushGit** [comment]
Stun has diminishing returns as well.
